src/CacheManager.Logging.SeriLog/SeriLogger.cs

```diff
@@ -0,0 +1,67 @@
+﻿using System;
+
+using cachelog = CacheManager.Core.Logging;
+using Serilog;
+using Serilog.Events;
+
+namespace CacheManager.Logging.SeriLog
+{
+    public class SeriLogger : cachelog.ILogger
+    {
+        private ILogger _logger;
+
+        public SeriLogger()
+        {
+            _logger = Serilog.Log.Logger;
+        }
+
+        public SeriLogger(string categoryName) : this()
+        {
+            SetCategoryName(categoryName);
+        }
+
+        public ILogger SetCategoryName(string name)
+        {
+            return _logger.ForContext(Serilog.Core.Constants.SourceContextPropertyName, name);
+        }
+
+        public IDisposable BeginScope(object state)
+        {
+            return null;
+        }
+
+        public bool IsEnabled(cachelog.LogLevel logLevel)
+        {
+            return _logger.IsEnabled(Convert(logLevel));
+        }
+
+        public void Log(cachelog.LogLevel logLevel, int eventId, object message, Exception exception)
+        {
+            _logger.Write(Convert(logLevel), exception, "backplane updating {0}:{1}", eventId, message);
```

Comment:
this was because the json output in logstash was coming out just as "{0}:{1}" which just got a little annoying. but ill remove the comment